packages/amplication-server/src/core/build/build.service.ts

```diff
@@ -639,7 +639,7 @@ export class BuildService {
     const level = ACTION_LOG_LEVEL[info[LEVEL]];
```

Comment:
May I remove this `level` defined here.

Reply:
What is calling this `createLog` method? Have you tested this change locally?
I'd suggest to try following the flow in order to fully understand where the change should be. This method doesn't seems to be called and we should probably delete it.